src/shared/helpers.jsx

```diff
@@ -0,0 +1,47 @@
+import { ButtonsAdmin } from '../components/Buttons/ButtonsAdmin/ButtonsAdmin';
+import { ButtonsStatusUpdate } from '../components/Buttons/ButtonsStatusUpdate/ButtonsStatusUpdate';
+
+export async function filterMembers(items) {
+  const users = await items;
```

Comment:
why do you `await` variable ? you already should have resolved result

Reply:
Yes, deleted async await here.